### PR TITLE
add probe to get the failure rate of the  services

### PR DESCRIPTION
**Pull request by luismenco**
add first version for dynatrace

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    'License :: OSI Approved :: Apache Software License',
+    'Programming Language :: Python',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.5',
+    'Programming Language :: Python :: 3.6',
+    'Programming Language :: Python :: Implementation',
+    'Programming Language :: Python :: Implementation :: CPython'
+]
+author = 'chaostoolkit Team'
+author_email = 'sh@defuze.org'
```
Comment (Lawouach) [inline setup.py:33]: I'll change this to contact@chaostoolkit.org

But you can also use your own email if you prefer since you authored the code?

**Review by Lawouach — APPROVED** [review]
LGTM

**Comment by Lawouach** [comment]
This is great @luismenco thank you so much!

File: setup.py (hunk 1 above)
Comment (luismenco) [reply setup.py:33]: Hello @Lawouach,
the email contact@chaostoolkit.org is perfect.

**Pull request merged**